src/StockportWebapp/wwwroot/assets/sass/stockportgov/modules/_navCard.scss

```diff
@@ -109,10 +109,17 @@
     &--force-border-top {
         @media #{$mobile, $tablet} {
```

Comment:
if $tablet is specified, $mobile is not needed as the styles will automatically be applied to both devices